content/en/developers/faq/data-collection-resolution-retention.md

```diff
@@ -20,6 +20,10 @@ Find below a summary of Datadog data collection, resolution, and retention:
 | Trace Events (Unsampled)     | Datadog Agent + tracing library + enabled logs                        | Real time           | 1 micro sec        | 2 Weeks                                 | APM + Logs       |
 | Logs                         | Datadog Agent + Logs, 3rd party log collectors, or API                | Real time           | 1 micro sec        | Plan                                    | Logs             |
 | API Test metrics             | Datadog Synthetics application                                        | 1 min               | 1 min              | 15 months                               | Synthetics       |
+| API Test results             | Datadog Synthetics application                                        | User-defined        | 1 min              | Result seen in the UI by user: 15 months <br/> Failed result not seen in the UI by user: 1 month <br/> Successful result not seen in the UI by user: 15 days                               | Synthetics       |
 | Browser Test metrics         | Datadog Synthetics application                                        | 5 mins              | 5 min              | 15 months                               | Synthetics       |
+| Browser Test results         | Datadog Synthetics application                                        | User-defined        | 5 min              | Result seen in the UI by user: 15 months <br/> Failed result not seen in the UI by user: 1 month <br/> Successful result not seen in the UI by user: 15 days                               | Synthetics       |
+| Uptime data lines            | Datadog Synthetics application                                        |                     |                    | 3 months                                | Synthetics       |
```

Comment:
Not sure about why we wrote `Uptime data lines`? 
Collection method is not only synthetics, but more generally all the monitors (including synthetics), and also metrics I believe.
@meghanelizabeth will probably be able to confirm that and the rest of the columns for uptime data :)

Reply:
currently synthetics does it's own `uptime` calculation, eventually this will move to the SLO product, but really @tanguylebarzic should probably review this.

Reply:
Thanks for the ping @platinummonkey. For this, we are currently using the Cassandra transitions table to compute the uptime, and indeed it would be great for us to stop computing our own and move the SLO product as soon as it's deemed stable enough, so in any case the retention will be controlled by the alerting teams. I would probably remove this line until we all agree on the retention the alerting teams will be able to provide.